backend/src/main/java/com/sscatalog/specialistsservicescatalog/services/SpecialistsService.java

```diff
@@ -1,17 +1,35 @@
 package com.sscatalog.specialistsservicescatalog.services;
 
+import com.sscatalog.specialistsservicescatalog.dtos.AddOfferedServiceRequest;
+import com.sscatalog.specialistsservicescatalog.dtos.ServiceDto;
+import com.sscatalog.specialistsservicescatalog.dtos.OfferedServiceDto;
+import com.sscatalog.specialistsservicescatalog.entities.OfferedService;
+import com.sscatalog.specialistsservicescatalog.entities.Service;
 import com.sscatalog.specialistsservicescatalog.entities.Specialist;
 import com.sscatalog.specialistsservicescatalog.entities.User;
+import com.sscatalog.specialistsservicescatalog.exceptions.ApiException;
+import com.sscatalog.specialistsservicescatalog.repositories.OfferedServiceRepository;
+import com.sscatalog.specialistsservicescatalog.repositories.ServiceRepository;
 import com.sscatalog.specialistsservicescatalog.repositories.SpecialistRepository;
-import org.springframework.stereotype.Service;
 
-@Service
+import java.util.List;
+import java.util.stream.Collectors;
+
+@org.springframework.stereotype.Service
```

Comment:
Why do not you use just @Service annotation here ?

Reply:
Because my entity class for service has the same name